adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -451,7 +451,7 @@ private AuthenticationConstants() { }
          * Signature info for Azure authenticator app that installs authenticator
          * component.
          */
-        public static final String AZURE_AUTHENTICATOR_APP_SIGNATURE = "ho040S3ffZkmxqtQrSwpTVOn9r0=";
```

Comment:
The authenticator signature is hard-coded which should be the same with the sign of prod build of Authenticator. The change you are trying to push will effect all other dev who are using our library. So it could not change here.